air-quality-backend/src/air_quality/api/measurements_controller.py

```diff
@@ -27,3 +31,21 @@ async def get_measurements(
 
     log.info(f"Responding with {len(db_results)} results")
     return map_measurements(db_results)
+
+
+@router.get("/air-pollutant/measurements/summary")
+async def get_measurements_summary(
+    measurement_base_time: datetime,
+    measurement_time_range: int,
+    location_type: AirQualityLocationType,
+):
+    log.info(
+        "Fetching measurements aggregated around {} (+/- {}) for {}".format(
+            measurement_base_time, measurement_time_range, location_type
```

Comment:
this string is formatted differently to the one on line 50 below which uses interpolation, is there a reason?

Reply:
It was for linting! The auto formatter doesn't like the other way when it's too long.